atc/metric/emit.go

```diff
@@ -132,13 +141,19 @@ func emit(logger lager.Logger, event Event) {
 
 	select {
 	case emissions <- eventEmission{logger: logger, event: event}:
+		logger.Debug("emit-event-write-to-channel", lager.Data{
```

Comment:
This on the other hand seems like it'd be a bit too noisy. :sweat_smile: Would you be OK with removing either this one or the above log?

(Also interesting that the above log line will run even with a `nil` emitter. Not sure if that's useful. I guess you'd notice it's not configured more quickly, but it also means those logs will show up even in places where metrics are undesired.)